app/env.template

```diff
@@ -0,0 +1,14 @@
+POSTGRES_PASSWORD
+POSTGRES_USER
+POSTGRES_DB
+POSTGRES_HOST
+POSTGRES_PORT
+ENVIRONMENT
+SECRET_KEY
+MINIO_ROOT_USER
+MINIO_ROOT_PASSWORD
+MINIO_ACCESS_KEY
+MINIO_SECRET_KEY
+AWS_STORAGE_BUCKET_NAME
+AWS_S3_ENDPOINT_URL
+API_KEY
```

Comment:
It should be with `=` and you can leave some defaults filled (ofc secrets empty)